apps/print/frontend/src/components/screen_wrapper.tsx

```diff
@@ -10,20 +10,30 @@ import {
   Main,
 } from '@votingworks/ui';
 import { Link, useRouteMatch } from 'react-router-dom';
+import { assertDefined } from '@votingworks/basics';
 import { Toolbar } from './toolbar';
 import { routeMap } from '../routes';
-import { getElectionDefinition } from '../api';
+import { getElectionRecord, getMachineConfig } from '../api';
 
 export function ScreenWrapper({
   children,
   authType,
 }: {
   children: React.ReactNode;
   authType: 'system_admin' | 'election_manager' | 'poll_worker';
-}): JSX.Element {
+}): JSX.Element | null {
   const currentRoute = useRouteMatch();
-  const getElectionDefinitionQuery = getElectionDefinition.useQuery();
-  const electionDefinition = getElectionDefinitionQuery.data;
+  const getElectionRecordQuery = getElectionRecord.useQuery();
+  const getMachineConfigQuery = getMachineConfig.useQuery();
+
+  if (!getElectionRecordQuery.isSuccess || !getMachineConfigQuery.isSuccess) {
+    return null;
+  }
+
+  const { electionDefinition, electionPackageHash } = assertDefined(
```

Comment:
Will we ever render this wrapper for unconfigured screens, which would result in `getElectionRecordQuery.data` being `null`?

Reply:
Good call, both good ideas. I like calling it `ConfiguredScreenWrapper` for clarity

Reply:
> for EMs should we include the stripped down wrapper, similar to Pollbook? Or should we just show the UnconfiguredScreen?

I think we should do whatever is simplest to implement. The empty sidebar is nice for consistency but I wouldn't go out of your way to build it out or special case anything.